NineChronicles.Headless/GraphTypes/FungibleAssetValueInputType.cs

```diff
@@ -40,5 +41,26 @@ public override object ParseDictionary(IDictionary<string, object?> value)
             return currency * (BigInteger)value["quantity"]!;
 #pragma warning restore CS0618
         }
+
+        public override IValue? ToAST(object value)
+        {
+            if (value is FungibleAssetValue fav)
+            {
+                var minters = new List<StringValue>();
+                if (fav.Currency.Minters is not null)
+                {
+                    minters.AddRange(fav.Currency.Minters.Select(minter => new StringValue(minter.ToString())));
+                }
+                return new ObjectValue(new List<ObjectField>
+                {
+                    new("quantity", new BigIntValue(fav.RawValue)),
+                    new("ticker", new StringValue(fav.Currency.Ticker)),
+                    new("decimalPlaces", new IntValue(fav.Currency.DecimalPlaces)),
+                    new("minters", new ListValue(minters)),
```

Comment:
Since the meanings of `minters is null` and `minters is List<> list && list.Length == 0` are different, shouldn't it be nullable when the `minters` is `null`?

Reply:
fix on https://github.com/planetarium/NineChronicles.Headless/pull/2083/commits/0dafd43358d5fcbce6fd6308ba20800b47f3e53b

before
```
    # The gas limit for Transaction.
    gasLimit: Long = 4
    maxGasPrice: FungibleAssetValueInputType = {
      quantity: 1000000000000000000
      ticker: "Mead"
      decimalPlaces: 18
      minters: []
    }
  ): ActionTxQuery!
```

fix
```
    # The gas limit for Transaction.
    gasLimit: Long = 4
    maxGasPrice: FungibleAssetValueInputType = {
      quantity: 1000000000000000000
      ticker: "Mead"
      decimalPlaces: 18
      minters: null
    }
  ): ActionTxQuery!
```